docs/how_to_run.md

```diff
@@ -118,6 +118,7 @@ available.
 
 -   Pushover Notifications
     -   `PUSHOVER_PUSH_USER`
+    -   `PUSHOVER_PUSH_TOKEN`
```

Comment:
This is mentioned in the `Optional Environment Variables` below since camply comes with a built-in token https://github.com/juftin/camply/blob/50e6f7919e331c5916ee63701b6a70805a2cf4bb/docs/how_to_run.md?plain=1#L151

Reply:
Ah, missed it. Sorry. However, I needed to supply my own API key using this flag -- the built-in token doesn't work for Pushover. Perhaps the max/month was reached.